pkg/tfvars/vsphere/vsphere.go

```diff
@@ -117,11 +124,18 @@ func TFVars(sources TFVarsSources) ([]byte, error) {
 // The datacenter could be reused but a folder could be
 // unique - the key then becomes a string that contains
 // both the datacenter name and the folder to be created.
-func createDatacenterFolderMap(infraID string, failureDomains []vtypes.FailureDomain) map[string]*folder {
+func createDatacenterFolderMap(infraID string, failureDomains []vtypes.FailureDomain, vcenters []vtypes.VCenter) (map[string]*folder, error) {
 	folders := make(map[string]*folder)
+	soap, _, cleanup, err := vsphere.CreateVSphereClients(context.TODO(), vcenters[0].Server, vcenters[0].Username, vcenters[0].Password)
```

Comment:
`vcenters` may be zero length 

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/6412/pull-ci-openshift-installer-master-e2e-vsphere-ovn/1574388049691807744